src/components/SummaryPanel/SummaryPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import ServiceInfoBadge from '../../pages/ServiceDetails/ServiceInfo/ServiceInfoBadge';
+import { ErrorRatePieChart } from './ErrorRatePieChart';
+import { RpsChart } from './RpsChart';
+
+type SummaryPanelPropType = {
+  namespace: string;
+  service: string;
+  data: any;
+};
+
+export default class SummaryPanel extends React.Component<SummaryPanelPropType, {}> {
+  static readonly panelStyle = {
+    position: 'absolute' as 'absolute',
+    width: '25em',
+    bottom: 0,
+    top: 0,
+    right: 0
+  };
+
+  render() {
+    const serviceHotLink = (
+      <a href={`../namespaces/${this.props.namespace}/services/${this.props.service}`}>{this.props.service}</a>
+    );
+
+    return (
+      <div className="panel panel-default" style={SummaryPanel.panelStyle}>
+        <div className="panel-heading">Service: {serviceHotLink} (v5)</div>
+        <div className="panel-body">
+          <p>
+            <strong>Labels:</strong>
+            <br />
+            {this.renderLabels()}
+          </p>
+          <hr />
+          <div style={{ fontSize: '1.2em' }}>
+            {this.renderIncomingRpsChart()}
+            {this.renderOutgoingRpsChart()}
+            <ErrorRatePieChart percentError={10} />
```

Comment:
Here's where the data won't be a percentage, it will be a rate (error per second or something like that).

Reply:
OK, so either we require the caller to calculate the percentError here or we add a "failureRate" and "totalRate" attributes here. I suspect we want to just leave this the way it is and expect the caller to pass in the properly calculated percentError. Yes? In which case we can accept this PR as is.??